src/include/duckdb/main/extension_entries.hpp

```diff
@@ -216,27 +216,27 @@ static constexpr ExtensionFunctionEntry EXTENSION_FUNCTIONS[] = {
 }; // END_OF_EXTENSION_FUNCTIONS
 
 static constexpr ExtensionEntry EXTENSION_SETTINGS[] = {
```

Comment:
This was created by using the generate script?

Reply:
Yes, why?

Reply:
I meant, what looks off to you?

Reply:
Just checking, because the header order was switched up, but no changes to the python script were pushed

Reply:
I think they have been updated in sync but they have not been generated for a while. Also more differences where to be found before https://github.com/duckdb/duckdb/pull/10948 was merged